app/javascript/blacklight/checkbox_submit.js

```diff
@@ -37,7 +37,11 @@ export default class CheckboxSubmit {
       this.labelTarget.removeAttribute('disabled')
       this.checkboxTarget.removeAttribute('disabled')
       this.updateStateFor(!this.checked)
-      if (this.bookmarkCounter()) this.bookmarkCounter().innerHTML = json.bookmarks.count
+      if (this.bookmarkCounter()) {
```

Comment:
I think we can remove this `if`, since `querySelectorAll` returns a truthy value whether it finds anything or not.

Reply:
Ah yes, that's true, I've made the changes